models/rag__unified_document.sql

```diff
@@ -6,8 +6,7 @@
         cluster_by = ['unique_id'],
```

Comment:
```suggestion
        cluster_by = ['update_date'],
```
For Snowflake's benefit, this would make more sense to be `update_date` to since we filter on that.